games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="3D open-source go-kart racing game"
+DESCRIPTION="3D open-source go-kart racing game with an extensive variety of characters, tracks, and modes to play."
```

Comment:
Maybe a more extensive description could look better. I can take on that.

Reply:
OK